### PR TITLE
Trigger versioning

### PR DESCRIPTION
**Pull request by schoren**
This PR updates the server model so all the test information is versioned within a single table (test). Also, updates the FE so it retrives the correct test details/trigger for the version corresponding to the selected test run.

## Fixes

- #1123 

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [x] added a test

### CODE REVIEW

File: server/testdb/tests.go [hunk 1]
```diff
@@ -47,16 +47,12 @@ func (td *postgresDB) CreateTest(ctx context.Context, test model.Test) (model.Te
 	if err != nil {
 		return model.Test{}, fmt.Errorf("encoding error: %w", err)
 	}
+	fmt.Println(string(b))
```
Comment (cescoferraro) [inline server/testdb/tests.go:50]: we can remove this

File: tracetesting/definitions/test_delete.yml [hunk 2]
```diff
@@ -20,4 +20,4 @@ specs:
 - selector: span[name = "exec DELETE"]
   assertions:
   # must delete test, definition, and runs. That's why we need 3 deletes
```
Comment (cescoferraro) [inline tracetesting/definitions/test_delete.yml:22]: we can edit the comment here to match whats being asserted

**Review by cescoferraro — APPROVED** [review]
small things I found. The FE code look fine!

**Review by xoscar — APPROVED** [review]
LGTM!

**Review by jorgeepc — APPROVED** [review]

**Pull request merged**